app/_src/gateway/support/browser.md

```diff
@@ -3,26 +3,12 @@ title: Browser Support
 breadcrumb: Browser
 ---
 
-[Edge](https://blogs.windows.com/msedgedev/2021/07/15/opt-in-extended-stable-release-cycle/), [Chrome](https://www.chromium.org/chrome-release-channels/) and [Firefox](https://support.mozilla.org/en-US/kb/switch-to-firefox-extended-support-release-esr) release a new major version every 4 weeks, with extended support available for 8 weeks.
+Kong supports the two latest stable versions and any extended support versions of the following "evergreen" desktop browsers:
 
-Kong supports N-1 versions of Edge, Chrome, Firefox and Safari on desktop plus any extended support versions.
+{% include_cached gateway-support-browsers.html %}
+
+[Chrome](https://www.chromium.org/chrome-release-channels/), 
+[Firefox](https://support.mozilla.org/en-US/kb/switch-to-firefox-extended-support-release-esr), and 
+[Edge](https://blogs.windows.com/msedgedev/2021/07/15/opt-in-extended-stable-release-cycle/) release a new major version every 4 weeks, 
+with extended support available for 8 weeks.
```

Comment:
Is the extended support on the browser's end or ours? Ex. is it Kong that offers extended support for 8 weeks?

Reply:
It's the browser's support.

Reply:
I looked at this some more and considered flipping this sentence back up to the top, but it doesn't mention Safari so that part is confusing. There is no doc for Safari fyi, because Apple.